recurring_contract/models/move.py

```diff
@@ -151,83 +149,65 @@ def update_open_invoices(self, updt_val):
                 else:
                     invoice.button_cancel()
 
-    def _build_invoice_data(self, contract=False, invoice_date=False, ref=False, pay_mode_id=False,
+    def _build_invoice_data(self, contracts=False, invoice_date=False, ref=False, pay_mode_id=False,
```

Comment:
```suggestion
    def _build_invoices_data(self, contracts=False, invoice_date=False, ref=False, pay_mode_id=False,
```
Name should be adapted to new behaviour